### PR TITLE
[INLONG-393][Sort] Add usage document for MySQL Extract Node, Elasticsearch Load Node

### LINKED ISSUE #393: [Sort] Add usage document for MySQL Extract Node, Elasticsearch Load Node
**Is your feature request related to a problem? Please describe.**

MySql Extract Node, Elasticsearch Load Node is missing some usage documentation here.

**Describe the solution you'd like**

Add usage document for MySQL Extract Node, Elasticsearch Load Node.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yunqingmoswu**
Fixes #393 

### Motivation

Add usage document for MySQL Extract Node, Elasticsearch Load Node

### Modifications

The usage document for MySQL Extract Node, Elasticsearch Load Node will be added.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by GanfengTan — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mysql-cdc.md [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+```
+<dependency>
+    <groupId>org.apache.inlong</groupId>
+    <artifactId>sort-connector-mysql-cdc</artifactId>
+    <!-- 选择适合你的 Application 的版本 -->
+    <version>inlong_version</version>
+</dependency>
+```
+
+连接 MySQL 数据库还需要 MySQL 驱动程序依赖项。请下载[mysql-connector-java-8.0.21.jar](https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.21/mysql-connector-java-8.0.21. jar) 并将其放入`FLINK_HOME/lib/`。
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mysql-cdc.md:31]: The download path was incorrect, there is an extra space.

Comment (yunqingmoswu) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mysql-cdc.md]: thanks.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/elasticsearch.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+连接器可以工作在 Upsert 模式，使用 DDL 中定义的主键与外部系统交换 UPDATE/DELETE 消息。
+
+如果 DDL 中没有定义主键，那么连接器只能工作在 Append 模式，只能与外部系统交换 INSERT 消息。
+
+## 支持的版本
+
+| Load 节点                             | 版本                                                 | 
+|-------------------------------------|----------------------------------------------------|
+| [elasticsearch](./elasticsearch.md) | [ElasticSearch](https://www.elastic.co/): 6.x, 7.x |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/elasticsearch.md:18]: The `ElasticSearch` should be `Elasticsearch`.

Comment (yunqingmoswu) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/elasticsearch.md:18]: done

**Review by healchow — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**